lightningd/closing/closing.c

```diff
@@ -0,0 +1,366 @@
+#include <bitcoin/script.h>
+#include <close_tx.h>
+#include <daemon/htlc.h>
+#include <errno.h>
+#include <inttypes.h>
+#include <lightningd/closing/gen_closing_wire.h>
+#include <lightningd/crypto_sync.h>
+#include <lightningd/debug.h>
+#include <lightningd/derive_basepoints.h>
+#include <lightningd/status.h>
+#include <lightningd/subd.h>
+#include <signal.h>
+#include <stdio.h>
+#include <type_to_string.h>
+#include <utils.h>
+#include <version.h>
+#include <wire/peer_wire.h>
+#include <wire/wire_sync.h>
+
+/* stdin == requests, 3 == peer, 4 = gossip */
+#define REQ_FD STDIN_FILENO
+#define PEER_FD 3
```

Comment:
It keeps confusing me that we send `fds` but then don't read them :-) Maybe I'll just add a short note after parsing the init message to remind my future self.